packages/lib-classifier/src/store/tasks/DataVisAnnotationTask.js

```diff
@@ -1,8 +1,9 @@
 import { types } from 'mobx-state-tree'
 import Task from './Task'
 import { Graph2dRangeXTool } from './dataVisTools'
+import { DataVisAnnotation } from '../annotations'
 
-const DataVisAnnotation = types.model('DataVisAnnotation', {
+const DataVisTask = types.model('DataVisTask', {
```

Comment:
Since we've been talking about naming, how about `DataVisTaskModel` here?

Reply:
Good point.